lib/auth/grpcserver.go

```diff
@@ -48,8 +45,10 @@ import (
 	"google.golang.org/grpc/peer"
 	"google.golang.org/grpc/status"
 
-	// Register gzip compressor for gRPC.
-	_ "google.golang.org/grpc/encoding/gzip"
+	apievents "github.com/gravitational/teleport/api/types/events"
+	wanlib "github.com/gravitational/teleport/lib/auth/webauthn"
+
+	_ "google.golang.org/grpc/encoding/gzip" // gzip compressor for gRPC.
```

Comment:
You don't have to do anything else to enable gzip? How does it know what to hook into?

Reply:
I just moved it around, but I assume an `init` function registers a handler for the GZIP Content-Encoding in the base grpc package.